.npmignore

```diff
@@ -0,0 +1 @@
+test/
```

Comment:
I didn't notice before this was merged.  This may cause issues.  Having this file means .gitignore is ignored when packing and we might end up deploying cruft.  Need to either add all the default and custom gitignore files here too or use "files" field to just list the handful of files that should be distributed.